src/library_pthread.js

```diff
@@ -96,8 +96,30 @@ var LibraryPThread = {
         PThread.allocateUnusedWorker();
       }
 #endif
+      // Spawn a dedicated worker for passing messages between threads. Instead
+      // of having each thread hold a message port for every other thread, they
+      // can just send messages with a `targetThread` property to the broker and
+      // the broker will forward the message to the intended recipient.
+      // Alternatively, the main thread could play this role, but then messages
+      // could be held up while the main thread is busy with other work.
+      PThread.proxyBroker = new Worker(new URL(
+          'data:text/javascript;base64,' + btoa(
+#if ENVIRONMENT_MAY_BE_NODE
+              (ENVIRONMENT_IS_NODE ?
+               `
+(await import('node:worker_threads'))
```

Comment:
Unfortunately I can't put this header in proxy_worker.js because Chrome complains about the existence of the `import` token, even if it isn't executed. Ideas for cleaner workarounds very welcome.